File: hw/carvajal/u1/hw04MultiplicationTable/MultiplicationTable.js
let number = parseInt(prompt("Which multiplication table would you like to see? "));

console.log(`\ntable of ${number}\n`);
console.log("Number| Result");
console.log("-------------------");

for (let i = 1; i <= 10; i++) {
  console.log(`${number.toString().padStart(6)} x ${i.toString().padEnd(2)} = ${(number * i).toString().padEnd(3)}`);
}

File: hw/carvajal/u1/hw04MultiplicationTable/MultiplicationTable.py
number = int(input("Which multiplication table would you like to see? "))

print(f"\nTabla del {number}\n")
print("Number| Result")
print("-------------------")

for i in range(1, 11):
    print(f"{number:>6} x {i:<2} = {number * i:<3}")